src/main/java/uk/ac/cam/cl/dtg/util/LogUtils.java

```diff
@@ -1,11 +1,50 @@
 package uk.ac.cam.cl.dtg.util;
 
+import java.util.Map;
+
 import static org.apache.commons.text.StringEscapeUtils.escapeJava;
 
 public final class LogUtils {
     private LogUtils() { }
 
+    /**
+     * Sanitise a value to be logged.
+     * The called function will escape any characters outside the unicode range of 32-127 (Latin and ASCII alphabet,
+     * symbols and digits).
+     *
+     * @param value the String to be sanitised
+     * @return the sanitised String
+     */
     public static String sanitiseLogValue(final String value) {
         return escapeJava(value);
     }
+
+    /**
+     * Sanitise a value to be logged. This value should be provided by internal sources such as admin accounts or
+     * platform infrastructure functions and thus clean, but it may be necessary to guard against malicious forged requests.
+     *
+     * @param value the String to be sanitised
+     * @return the sanitised String
+     */
+    public static String sanitiseInternalLogValue(final String value) {
```

Comment:
Why do we need this method and `sanitiseUserLogValue`, when they both do exactly the same thing as `sanitiseLogValue`?

Also, the tests are missing for all these new methods.

Reply:
Removed one of the methods and updated the tests.
'External' log values can potentially be triggered by general users.
'Internal' log values are in code that's in the ETL and/or should require admin permissions to reach. They're being sanitised anyway just in case and to make CodeQL happy, but if a malicious user can trigger those, we may have bigger concerns than some extra text in the logs.
There's currently no difference in how they're handled, I just wanted to figure out and make clear which was which .